walkthroughs/1-integrating-event-and-api-driven-apps/walkthrough.adoc

```diff
@@ -120,7 +163,7 @@ Verify that you followed each step in the procedure above.  If you are still hav
 [.integr8ly-docs-header]
 === Creating AMQ connection in Red Hat Fuse Online
 
-To receive messages from AMQ, you create a connection in Red Hat Fuse Online.
+To receive messages, you create a connection in Red Hat Fuse Online using the AMQ connector.
```

Comment:
"To receive messages, you create a connection" => "To receive messages placed on the queue by the Node.js webapp"

Not really necessary, but might help to include the full context.

Reply:
To allow Fuse Online to consume messages placed on the queue by the Node.js webapp, you create a connection in Red Hat Fuse Online.